pkg/modules/qpdf/qpdf.go

```diff
@@ -101,6 +101,27 @@ func (engine *QPdf) Merge(ctx context.Context, logger *zap.Logger, inputPaths []
 	return fmt.Errorf("merge PDFs with QPDF: %w", err)
 }
 
+// Flatten merges annotation appearances with page content, deleting the original annotations.
+func (engine *QPdf) Flatten(ctx context.Context, logger *zap.Logger, inputPath, outputPath string) error {
+	var args []string
+	args = append(args, "--generate-appearances")
+	args = append(args, "--flatten-annotations=all")
+	args = append(args, inputPath)
+	args = append(args, outputPath)
```

Comment:
I wonder if it would make more sense to use the flag [--replace-input](https://qpdf.readthedocs.io/en/stable/cli.html#option-replace-input) instead of creating a new file 🤔 

```suggestion
func (engine *QPdf) Flatten(ctx context.Context, logger *zap.Logger, inputPath string) error {
	var args []string
	args = append(args, "--generate-appearances")
	args = append(args, "--flatten-annotations=all")
	args = append(args, "--replace-input")
	args = append(args, inputPath)
```

Reply:
makes sense, will do